setup.py

```diff
@@ -12,5 +17,6 @@
   author='balloob, ggravlingen',
   author_email='no@email.com',
   url='https://github.com/ggravlingen/pytradfri',
-  download_url=DOWNLOAD_URL
+  download_url=DOWNLOAD_URL,
+  install_requires=REQUIRES,
```

Comment:
Actually, let's instead no longer import the factories into `__init__.py` but instead have users import from `pytradfri.aiocoap` etc.